backend/game/dice/dice_resolver.py

```diff
@@ -61,9 +62,13 @@ def resolve_all_other_dice(dice_counter, player):
 
 def card_based_dice_actions(dice_counter, player, other_players):
     if dice_counter[DieValue.ONE] >= 3 and player.has_instance_of_card(Gourmet()):
-        Gourmet.special_effect(player, other_players)
+        Gourmet().special_effect(player, other_players)
     if dice_counter[DieValue.ONE] >= 1 and dice_counter[DieValue.TWO] >= 1 and dice_counter[DieValue.THREE] >= 1 and player.has_instance_of_card(Omnivore()):
-        Omnivore.special_effect(player, other_players)
+        Omnivore().special_effect(player, other_players)
+    if dice_counter[DieValue.ONE] >= 1 and dice_counter[DieValue.TWO] >= 1 and dice_counter[DieValue.THREE] >= 1 and \
```

Comment:
Could also be done at the dice level if you want to avoid all the if >=1 statements:

`goal = [DieValue.ONE, DieValue.TWO, DieValue.THREE, DieValue.HEAL, DieValue.ATTACK, DieValue.ENERGY]`
`if all(elem in player_rolled_dice for elem in goal):`